public/src/components/lab/LabResult.tsx

```diff
@@ -35,8 +35,8 @@ export class LabResult extends React.Component<ILabResult, ILabResultState> {
             approveButton = <div className="btn lab-btn approve-btn"> <button type="button"
                 id="approve"
                 className={this.setButtonColor("approve")}
-                onClick={this.props.isApproved ?
-                    () => { console.log("Already approved"); } : () => this.approve()}>
+                title={this.setTooltip()}
+                onClick={() => this.approve()}>
                      {this.setButtonString("approve")} </button> </div>;
```

Comment:
I would also end the </button... on a new line.

Reply:
Fixed.